tests/cephadm/test_label_no_schedule.py

```diff
@@ -0,0 +1,73 @@
+from json import loads
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class LabelNoScheduleError(Exception):
+    pass
+
+
+def adding_label(instance, node):
+    """
+    Method add _no_schedule label in the node
+    Args:
+        instance: cluster instance
+        node: cluster node
+    Returns:
+        boolean
+    """
+    log.info(f"Adding '__no_schedule' label in {node}")
+    cmd = f"cephadm shell ceph orch host label add {node} _no_schedule"
+    out, err = instance.installer.exec_command(cmd, sudo=True)
+    if "Added" in out:
+        return 0
+    raise LabelNoScheduleError("Failed to add _no_schedule label to node")
+
+
+def check_daemons(instance, node):
+    """
+    Method checks all the deamons in node
+    Args:
+        node: cluster node
+    Returns:
+        boolean
+    """
+    log.info(f"Checking all the deamons in {node}")
+    cmd = f"cephadm shell ceph orch ps {node} --format json"
```

Comment:
Same as above add this methods in `cli.ceph.orch` lib

Reply:
Updated

Reply:
@MohitBis Move add this `ps` command in PR https://github.com/red-hat-storage/cephci/pull/2009/files

Reply:
@MohitBis add this method in module - https://github.com/red-hat-storage/cephci/blob/master/cli/ceph/orch/orch.py

Reply:
@vamahaja 
Method added to orch.py
https://github.com/red-hat-storage/cephci/pull/2009/files#diff-775b7df2434d2d775287b5a432b25dbdabedf3db60be80982b743d2fb8f5a400R77